flexget/plugins/output/qbittorrent.py

```diff
@@ -60,10 +61,20 @@ def connect(self, config):
         log.debug('Successfully connected to qBittorrent')
         self.connected = True
 
-    def add_torrent(self, data):
+    def add_torrent_file(self, filepath, data):
         if not self.connected:
             raise plugin.PluginError('Not connected.')
         multipart_data = {k: (None, v) for k, v in data.items()}
+        with open(filepath, 'rb') as f:
+            multipart_data['torrents'] = f
+            self.session.post(self.url + '/command/upload', files=multipart_data)
+        log.debug('Added task to qBittorrent')
```

Comment:
You don't add a task, you add entries. The debug message is pointless as it provides no information.